addons/sale_mrp/tests/test_sale_mrp_procurement.py

```diff
@@ -1,11 +1,9 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-import time
 
+from odoo import Command
```

Comment:
```suggestion
from odoo.fields import Command
```

Reply:
`Command` is imported from Odoo in various files in our project. I'd rather avoid changing the line in case it would have an impact on the file.